blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/server/model/BbServerUser.java

```diff
@@ -14,14 +18,25 @@
     private final String displayName;
     private final String slug;
     private final String emailAddress;
+    private final String avatar;
 
     @JsonCreator
     public BbServerUser(@Nonnull @JsonProperty("name") String name, @Nonnull @JsonProperty("displayName") String displayName,
-                        @Nonnull @JsonProperty("slug") String slug, @Nonnull @JsonProperty("emailAddress") String emailAddress) {
+                        @Nonnull @JsonProperty("slug") String slug, @Nonnull @JsonProperty("emailAddress") String emailAddress, @JsonProperty("links") Map<String, List<Map<String,String>>> links) {
         this.name = name;
         this.displayName = displayName;
         this.slug = slug;
         this.emailAddress = emailAddress;
+        List<Map<String,String>> hrefs = links.get("self");
+        String a =null;
```

Comment:
Perhaps `avatar` instead of `a`? Sorry again to be picky about variable names...